frontend/src/components/match/MatchError.jsx

```diff
@@ -0,0 +1,31 @@
+import {
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+} from '@chakra-ui/react'
+
+const MatchError = ({ onClose, handleRetry }) => {
```

Comment:
Match Error/Match timeout components etc seems like a repeat with only changes to the header, perhaps can consolidate them in the future

Reply:
Agree, the main difference is the button, but will see how we can combinethese similar components